about.html

```diff
@@ -0,0 +1,189 @@
+        </li>
+        <li class="nav-item">
+        <a class="nav-link mt-1" href="#"><i class="bi bi-twitter"></i></a>
+        </li>
+        <li class="nav-item">
+        <a class="nav-link" href="#">English</a>
+        </li>
+        <li class="nav-item">
+        <a class="nav-link" href="#">My Page</a>
+        </li>
+        <li class="nav-item">
+        <a class="nav-link" href="#">Logout</a>
+        </li>
+    </ul>
+</div>
+
+    <!-- navbar -->
+    <nav class="navbar navbar-expand-lg">
+        <div class="container-lg">
+
+             <!-- Toggle button for mobile navbar -->
+             <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#main-nav" aria-controls="main-nav" aria-expanded="false" aria-label="Toggle Navigation">
+                <span onclick="ShowXicon()" class="navbar-toggler-icon" id="meneIcon"></span>
+                <span onclick="removeXiocn()" class="bi bi-x" id="Xicon"></span>
+            </button>
+            
+            <a href="#intro" class="navbar-brand">
+                <span class="logo">
+                  <a href="index.html"><img class="img-fluid logo" src="assets/images/logo/techlife.png" alt=""></a> 
+                </span>
+            </a>
+
+           
+            <!-- Navbar links -->
+            <div class="collapse navbar-collapse justify-content-end pe-5 pt-1 align-center nav-menu" id="main-nav">
+                <ul class="navbar-nav">
+                    <li class="nav-item pt-1">
+                        <a class="nav-link" href="about.html">About</a>
+                    </li>
+                    <li class="nav-item pt-1">
+                        <a class="nav-link" href="#">Program</a>
+                    </li>
+                    <li class="nav-item pt-1">
+                        <a class="nav-link" href="#">Join</a>
+                    </li>
+                    <li class="nav-item pt-1">
+                        <a class="nav-link" href="#">Sponsor</a>
+                    </li>
+                    <li class="nav-item pt-1">
+                        <a class="nav-link" href="#">News</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="#">CC Campaign</a>
+                    </li>
+                </ul>
+            </div>
+        </div>
+    </nav>
+
+    <!-- About main page  -->
+    <section id="about-intro">
+        <div class="container-lg pt-5 background-gradient">
+            <div class="row justify-content-center text-center align-items-center">
+                <div class="col-md-8">
+                    <h1 class="col-md-8 offset-md-2">
+                        <div class="display-6 px-4 d-block d-md-none"> "Hello! Sharing World"</div>
+                        <div class="display-5 about-header">Techlife  Innovative  global summit 2023</div>
+                        <div class="line my-2"></div>
+                    </h1>
```

Comment:
- [ ]  As required for the project to stick as much as possible to the one form the Behance template, kindly ensure that you improve the `fonts(size, weight,...)` on your Hero text, `remove the text` "Hello sharing..." as it doesn't appear on the About page of the template, and check other details to correctly match the required template,....

Reply:
"Hello Sharing" text exists just in the mobile version in the Behance template, and it also just appears in mobile version